pages/design_documentation/cleaning.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cleaner
+last_updated: August 17, 2022
+toc: false
+permalink: cleaning.html
```

Comment:
Here we may want to have link named `cleaner.html` to not confuse it with cleaning in basic concepts.